config/initializers/secret_token.rb

```diff
@@ -5,3 +5,4 @@
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
 SeattlerbOrg::Application.config.secret_token = 'eff82d437b2886034bb8da23aa99d75499baeed3f0ab1864774fdfaf66b98a935a1bb54377ae05c927af8eecba325e9cba80aec3cb5818c95a8da71539184186'
+SeattlerbOrg::Application.config.secret_key_base = '55a89dc06da0cc081f107e6702d58ea8e0c94b78903af52b5a791dad348619b740a0ceb36d575f5dafca6b6a9ae79ee37c988e11b2d707d657550b70a9d1f3aa'
```

Comment:
Ya, that's right.  We shouldn't be checking these in.  It's OK for dev and test envs, but for production, we should be reading these from an environment variable on Heroku.